app/controllers/items_controller.rb

```diff
@@ -4,11 +4,40 @@ class ItemsController < ApplicationController
   include ItemsHelper
   include DorObjectHelper
   include ModsDisplay::ControllerExtension
-  before_filter :create_obj, :except => [:register, :open_bulk, :purge_object]
-  before_filter :forbid_modify, :only => [:add_collection, :set_collection, :remove_collection, :update_rights, :set_content_type, :tags, :tags_bulk, :source_id, :delete_file, :close_version, :open_version, :resource, :add_file, :replace_file, :update_attributes, :update_resource, :mods, :datastream_update ]
-  before_filter :forbid_view,   :only => [:get_preserved_file, :get_file]
-  before_filter :enforce_versioning, :only => [:add_collection, :set_collection, :remove_collection, :update_rights, :tags, :source_id, :set_source_id, :set_content_type, :set_rights]
-  after_filter  :save_and_reindex,   :only => [:add_collection, :set_collection, :remove_collection, :open_version, :close_version, :tags, :tags_bulk, :source_id, :set_rights, :set_content_type, :apply_apo_defaults]
+  before_filter :create_obj, :except => [
+    :open_bulk, :purge_object, :register,
+  ]
+  before_filter :allow_modify, :only => [
+    :add_collection, :set_collection, :remove_collection,
+    :add_file, :delete_file, :replace_file,
+    :close_version, :open_version,
+    :resource,
+    :source_id,
+    :set_content_type,
+    :tags, :tags_bulk,
+    :update_rights,
+    :update_attributes, :update_resource, :datastream_update,
+    :mods,
+  ]
+  before_filter :allow_view, :only => [
+   :get_file, :get_preserved_file,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.